src/doc/swagger.ts

```diff
@@ -2,7 +2,7 @@ import swaggerAutogen from 'swagger-autogen';
 
 import { metadata } from '../config/openapi';
 
-const outputFile = './openapi.json';
+const outputFile = './dist/doc/openapi.json';
```

Comment:
We have a command to do that => refer to README.md

Reply:
si je met pas ca, ca genere la doc a la racine du projet